src/localization/modalStrings.json

```diff
@@ -73,7 +73,10 @@
     "bluetooth_permission_comment": "Required to connect to bluetooth sensors",
     "permissions_intro_1": "The following permissions and services are required for cold chain functionality.",
     "permissions_intro_2": "If disabled, you can press an item to enable it.",
-    "version_incompatible": "Unable to import! This is version of {0} of mSupply mobile and the selected import file has a newer version of {1}"
+    "version_incompatible": "Unable to import! This is version of {0} of mSupply mobile and the selected import file has a newer version of {1}",
+    "pending_sg_customer_requisition": "The mobile store you are trying to connect to has {0} suggested customer requisitions. We cannot sync the store while these suggested customer requisitions exist.\n\nWe suggest you to log into mSupply desktop and resolve (confirm, finalise or delete) these customer requisitions. Please click the \"Cancel\" button and try again after the issue is resolved.\n\nIf you think these suggested requisitions are obsolete, please click the \"Delete\" button below and we will delete all of the pending suggested customer requisitions automatically for you. However remember that these suggested customer requisitions would not be recoverable once deleted.\n\nDo you want to continue?",
```

Comment:
Minor grammar suggestion: `We suggest you to log into mSupply desktop and resolve`
to
`We recommend logging into mSupply desktop to resolve`